### PR TITLE
Translation improvements

### PR DESCRIPTION
**Pull request by VeederLicht**
As with the Belgian (Flemish) translation, these sentences can use some improvements. Especially the 'Paniek aan het veroorzaken in de update matrix'. That sentence makes no sense at all and makes SolusOS look very childish.

### CODE REVIEW

File: po/nl_NL.po [hunk 1]
```diff
@@ -768,15 +768,15 @@ msgstr "Niet geïnstalleerd"
 
 #: ../solus_sc/updates_view.py:87
 msgid "Discombobulating update matrix"
-msgstr "Paniek aan het veroorzaken in de update matrix"
+msgstr "Updates aan het verzamelen"
```
Comment (DataDrake) [inline po/nl_NL.po:771]: It's not supposed to make sense. It's meant to be funny.

**Comment by DataDrake** [comment]
I fail to see how a single line that is only displayed as a way to pass time while the user is waiting for the index to be updated means the whole of Solus is suddenly childish.

Also, It's **Solus** not Solus *OS*. Solus OS hasn't existed for most of the last decade.

**Comment by VeederLicht** [comment]
Maybe childish wasn't the word i was looking for, 'unprofessional' might 
be more fitting.
The reason i take the time to address this (where the other 99.9% just 
shake their heads and move on) is because i would like to see more 
people adopt Linux based OS'es, especially those who use their computers 
professionally. To many of them Linux has the 'unprofessional' and 'for 
nerds only' reputation already, lines like those i mentioned only 
compound that reputation.

To put it in other words: wether you open up your distro to more people 
or keep it closed down 'for nerds only' is in your own hands.


---
-------------------------

Bryan T. Meyers schreef op 2020-02-21 22:53:

**Comment by JoshStrobl** [comment]
God forbid we add some personality into the software we develop. This isn't an improvement, it's a regression. Closing as such.

**Pull request closed without merging**